chromium_src/chrome/browser/ui/browser.h

```diff
@@ -60,6 +60,7 @@ class Browser : public content::WebContentsDelegate {
   struct CreateParams {
     explicit CreateParams(Profile* profile);
     CreateParams(Type type, Profile* profile);
+    CreateParams(Type type, Profile* profile, bool user_gesture) {}
```

Comment:
why is this here? We definitely don't want to implement it as a no-op either way